### PR TITLE
OT_35 is owner added to middlewares

### PR DESCRIPTION
**Pull request by rich27a**
isOwner added to middlewares

### CODE REVIEW

File: middlewares/verify-role.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const { response } = require("express");
+
+const isOwner = (req, res = response, next) => {
+
+    if(!req.user){
```
Comment (soyarielruiz) [inline middlewares/verify-role.js]: El user no viene en el request.

File: middlewares/verify-role.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const { response } = require("express");
+
+const isOwner = (req, res = response, next) => {
```
Comment (soyarielruiz) [inline middlewares/verify-role.js]: Pensar en POO

File: middlewares/verify-role.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    if(!req.user){
+        return res.status(500).json({
+            msg: `Token not verified`
+        })
+    }
+
+    const {id, role, name} = req.user;
```
Comment (soyarielruiz) [inline middlewares/verify-role.js]: Debe validarse los datos del token.

File: middlewares/verify-role.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  static async isOwner(req, res = response, next) {
+    const authToken = req.headers['authorization'];
+    if (!authToken) {
+      res.status(500).json({
+        msg: 'There is no token in request',
+      });
+    }
+
+    try {
+      const { email } = jwt.verify(authToken, process.env.SECRETORPRIVATEKEY);
```
Comment (soyarielruiz) [inline middlewares/verify-role.js]: Queda como deuda técnica usar esta funcionalidad
https://github.com/alkemyTech/OT291-server/pull/28/files#diff-4a04631fb36f911fb698151a6f7f5104456de78dc644b0f158fb6c6df9ea655bR19

**Review by soyarielruiz — APPROVED** [review]

File: middlewares/verify-role.js (hunk 4 above)
Comment (rich27a) [reply middlewares/verify-role.js]: Corregido

**Review by soyarielruiz — APPROVED** [review]

**Pull request merged**